sql/firefox_accounts_derived/fxa_log_content_performance_events_v1/query.sql

```diff
@@ -0,0 +1,26 @@
+WITH base AS (
+  SELECT
+    receiveTimestamp AS `timestamp`,
+    jsonPayload.event,
+    jsonPayload.flow_time,
+    jsonPayload.locale,
+    jsonPayload.useragent,
+    jsonPayload.country,
+    jsonPayload.entrypoint,
+    jsonPayload.flow_id,
+    jsonPayload.region,
+    jsonPayload.service,
+    jsonPayload.utm_campaign,
+    jsonPayload.utm_content,
+    jsonPayload.utm_medium,
+    jsonPayload.utm_source,
+  FROM
+    `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_content_19700101`
+)
+SELECT
+  *
+FROM
+  base
+WHERE
+  REGEXP_CONTAINS(event, r"^flow\.performance.[\w-]+")
```

Comment:
Not that I'm planning to make these table partitioned on `DATE(timestamp)` and clustered on `event`, so it will be efficient to filter to just a single event type.

Would it make sense to collapse performance events and "other" events into a single table? I could also provide a view that filters to just performance events if the point here is to deliver different tables for different audiences.